.travis.yml

```diff
@@ -5,6 +5,7 @@ install:
   - make reqs
 script:
   - make pep8
+  - make flake8 FLAKE8_OPTS=--exit-zero
```

Comment:
Think we want to remove FLAKE8_OPTS=--exit-zero so we actually do fail travis builds on flake8 failures.